pkgs/development/ocaml-modules/decoders-jsonaf/default.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildDunePackage,
+  decoders,
+  jsonaf,
```

```suggestion
  jsonaf ? null,
```

Otherwise evaluation breaks for legacy versions of OCaml.